it_hardware/demo/product_template.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_template_4" model="product.template" context="{'create_product_product': False}">
+        <field name="is_published" eval="True"/>
+    </record>
+    <record id="product_template_5" model="product.template" context="{'create_product_product': False}">
+        <field name="is_published" eval="True"/>
+    </record>
+    <record id="product_template_6" model="product.template" context="{'create_product_product': False}">
+        <field name="is_published" eval="True"/>
+    </record>
```

Comment:
Maybe we should remove the warranty products from the home page and the shop (but allow as optional products)